scold.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
   s.require_paths = %w(lib)
   s.extra_rdoc_files = %w(README.md)
   s.rubygems_version = "2.5.0"
-  s.add_runtime_dependency("rubocop", "~> 0.37")
+  s.add_runtime_dependency("rubocop", "~> 0.54")
```

Comment:
hound is at .51  https://github.com/houndci/linters/blob/master/Gemfile.lock#L79

also, changing this value doesn't really do much, projects that use the scold gem can install any rubocop gem up to but not including 1.0.

Reply:
Yeah, I just figured I'd bump it up to what was explicitly tested.